manifests/0000_30_cluster-api_11_deployment.yaml

```diff
@@ -38,6 +38,9 @@ spec:
         - containerPort: 9443
           name: webhook-server
           protocol: TCP
+        - containerPort: 8443
```

Comment:
This is the default? I wonder if we should be explicit anyway and add the flag?

Reply:
Yeah it's default. Can't hurt to be explicit, will update.